src/encoded/commands/deploy.py

```diff
@@ -715,6 +718,21 @@ def main():
         )
         for key, val in tags_dict.items():
             print(f"{key:28}:'{val}'")
+        PG_URI='postgresql:///encoded'
+        if run_vars['ENCD_BUILD_TYPE'] == 'app' or run_vars['ENCD_BUILD_TYPE'] == 'app-es':
+            PG_URI="postgresql://run_vars['ENCD_PG_IP']/encoded"
+        bin_build_cmd= "bin/buildout -c {ENCD_ROLE}.cfg buildout:es-ip={ENCD_ES_IP} buildout:es-port={ENCD_ES_PORT} buildout:pg-uri={PG_URI} buildout:fe-ip={ENCD_FE_IP} buildout:remote_indexing={ENCD_REMOTE_INDEXING} buildout:index_procs={ENCD_INDEX_PROCS} buildout:index_chunk_size={ENCD_INDEX_CHUNK_SIZE}".format(
```

Comment:
Does this obviate the need to put these variables in `env.yml`? https://github.com/ENCODE-DCC/encoded/blob/dev/cloud-config/template-parts/env.yml

Since they are templated into the ini file here it looks like the buildout run on the AMIs could be simplified: https://github.com/ENCODE-DCC/encoded/blob/dev/cloud-config/run-scripts/app-encd.sh#L41

Finally, beyond the scope of the ticket but I feel like some of these variables aren't that dynamic and should probably live statically in the config files.

Reply:
hmm.  I think they buildout vars should end up in the env.  Then we could do something like

[buildout:os.environ.get("SOME_VARIABLE") == "123"]

which I've never tested.  Or just access the env directly from the application.

Reply:
and yes some of the vars like ports have never changed so I'll make those static.